app/views/pages/secondary_skip/_form.html.erb

```diff
@@ -0,0 +1,36 @@
+<%= form_with(model: secondary_skip_input, url: submit_path) do |f| %>
+  <% if  secondary_skip_input&.errors.any? %>
+    <%= f.govuk_error_summary %>
+  <% end %>
+
+  <h1 class="govuk-heading-l">
+    <span class="govuk-caption-l"><%= t("secondary_skip.new.caption", page_position: secondary_skip_input.page.position) %></span>
+    <%= t("page_titles.new_secondary_skip", route_index: 2) %>
+  </h1>
```

Comment:
It probably makes sense to have the page heading in the page template rather in this partial, so that it and the page title can be changed at the same time easily